docs/contributing/index.mdx

```diff
@@ -1,18 +1,18 @@
 ---
-title: Contributing to Gatsby.js
+title: Gatsby.js में योगदान देना
 disableTableOfContents: true
 ---
 
 import EmailCaptureForm from "../../www/src/components/email-capture-form"
 
-Thanks for being interested in contributing! We're so glad you want to help!
+सहयोग देने मे रूचि रखने के लिए आपका धन्यवाद! हमे बेहद खुशी है की आप सहयोग देना चाहते है!
 
-Below you'll find guides on the Gatsby.js community, code of conduct, and how to get started contributing:
+नीचे आपको Gatsby.js समुदाय पर मार्गदर्शक, कोड ऑफ़ कंडक्ट, और सहयोग देना शुरू कैसे करें पर मदद मिलेगी:
 
-- [Community](/contributing/community/): Learn why you should contribute to Gatsby.js, the most effective ways to do so, and all about the benefits (did we mention the free swag?)
-- [Code of Conduct](/contributing/code-of-conduct/): Read about what we expect from everyone participating in Gatsby.js to make it the most friendly and welcoming community
-- [Gatsby Style Guide ](/contributing/gatsby-style-guide/): The art of contributing to Gatsby, a.k.a. the detailed requirements that will make it more likely your contribution is accepted with minimal changes
-- [How to Contribute ](/contributing/how-to-contribute/): How to get the most out of your Gatsby.js contributing experience, including GitHub management tips, setup instructions for docs and code contributions, and more
-- [RFC process](/contributing/rfc-process): Learn how the Gatsby.js team manages bigger changes, by way of a “Request-for-comment” process on GitHub
+- [समुदाय](/contributing/community/): जाने की आपको Gatsby.js को सहयोग क्यों देना चाहिए, ऐसा करने का सबसे प्रभावी तरीका क्या है और, सभी लाभों के बारे में (क्या हमने फ्री स्वाग का उल्लेख किया है?)
+- [कोड ऑफ़ कंडक्](/contributing/code-of-conduct/): पढ़िए की हम Gatsby.js मे भाग लेने वाले सभी लोगों से क्या उम्मीद रखते हैं ताकि यह सबसे अनुकूल और स्वागत करने वाला समुदाय बन पाए
+- [Gatsby की स्टाइल गाइड ](/contributing/gatsby-style-guide/): मे सहयोग देने की कला, जिसको डिटेल्ड रिक्वायरमेंट्स के नाम से भी जाना जाता है जो आपके सहयोग को काम से काम परिवर्तन के साथ भी स्वीकार होने की संभावना बढ़ा देता है
```

Comment:
`मे सहयोग देने की कला, जिसको डिटेल्ड रिक्वायरमेंट्स के नाम से भी जाना जाता है जो आपके सहयोग को काम से काम परिवर्तन के साथ भी स्वीकार होने की संभावना बढ़ा देता है` => `Gatsby में योगदान देने की कला, अर्थात् आपका योगदान संभावित परिवर्तनों के साथ स्वीकार कराने के लिए विस्तृत आवश्यकताएं जो इसे और अधिक संभव बनाती हैं`

How about this?